docs/content/configuration/csi_ug_config_create_vol_snapshotclass_topology.md

```diff
@@ -0,0 +1,48 @@
+# Creating a VolumeSnapshotClass with topology awareness
+
+When using the CSI Topology feature, different parameters must be taken into account when creating a VolumeSnapshotClass YAML file with specific `by_management_id` requirements. Use this information to help define a VolumeSnapshotClass that is topology aware and enables the creation and deletion of volume snapshots.
+
+**Note:** 
+  - For information and parameter definitions that are not related to topology awareness, be sure to see the information provided in [Creating a VolumeSnapshotClass](csi_ug_config_create_vol_snapshotclass.md), in addition to the current section.
+  
+  - IBM® FlashCopy® function is referred to as the more generic volume snapshots and cloning within this documentation set. Not all supported products use the FlashCopy function terminology.
+
+In order to enable creation and deletion of volume snapshots for your storage system, create a VolumeSnapshotClass YAML file, similar to the following `demo-volumesnapshotclass-config-secret.yaml`.
+
+When configuring the file, be sure to use the same array secret and array secret namespace as defined in [Creating a Secret with topology awareness](csi_ug_config_create_secret_topology.md).
+
+**Note:**
+  - The `snapshot_name_prefix` parameter is optional.
+  - The `CSI` prefix is added as default if not specified by the user.
```

Comment:
this is true only for svc

Reply:
also, was the maximum prefix length omitted here on purpose?

Reply:
consider removing notes that overlap with non-topology pages

Reply:
1. _this is true only for svc_ - Topology is currently for SVC only. Although we will probably add in more storage support in the future, it doesn't make sense to add this wording in now. we'll have to just do it with the release of topology for the other systems.
2. _also, was the maximum prefix length omitted here on purpose?_ - no, it was an accident and has now been added in.
3. _consider removing notes that overlap with non-topology pages_ - I removed anything that I thought made sense to remove but some things needed to be kept in for various documentation reasons.

DONE